orchagent/muxorch.cpp

```diff
@@ -1531,10 +1541,18 @@ MuxStateOrch::MuxStateOrch(DBConnector *db, const std::string& tableName) :
 
 void MuxStateOrch::updateMuxState(string portName, string muxState)
 {
+    /* Disable Pfc Watchdog for perfomance */
+    SWSS_LOG_INFO("Disabling PFCWD for Mux Switchover on %s", portName.c_str());
+    gPfcWdSwOrch->stopWdOnAllPorts();
+
```

Comment:
Why do we need to disable first, and then enable again? Is it better if we cherk the incoming `muxstate` first?
And same question here, why do we disable/enable watchdog on all ports?